From 8a07e4a43cca6fbd3476f16a42c187cc733d1706 Mon Sep 17 00:00:00 2001
From: Kai-Chuan Hsieh <kaichuan.hsieh@canonical.com>
Date: Tue, 10 Jan 2023 16:36:51 +0800
Subject: [PATCH] hwdb: Mark Dell platform accel sensor location to base

Dell would like to disable screen rotation for the platform eternally.
Mark the aceel sensor location base to disable it.

(cherry picked from commit 8d4f1337e89eee222ff2e138980681d64e25c053)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 3aa45b8289..df81942fc8 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -348,6 +348,7 @@ sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B0D:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B11:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C00:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C02:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C40:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C41:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C42:*
  ACCEL_LOCATION=base
